File: backend/src/main/java/com/example/ocrjizhang/backend/api/TransactionDto.java
package com.example.ocrjizhang.backend.api;

public record TransactionDto(
    long id,
    long userId,
    String type,
    long amountFen,
    long categoryId,
    String categoryName,
    String remark,
    String merchantName,
    long transactionTime,
    String source,
    long createdAt,
    long updatedAt
) {
}

File: app/src/main/java/com/example/ocrjizhang/data/remote/response/TransactionDto.kt
package com.example.ocrjizhang.data.remote.response

data class TransactionDto(
    val id: Long,
    val userId: Long,
    val type: String,
    val amountFen: Long,
    val accountId: Long?,
    val accountName: String?,
    val categoryId: Long,
    val categoryName: String,
    val remark: String?,
    val merchantName: String?,
    val transactionTime: Long,
    val source: String,
    val createdAt: Long,
    val updatedAt: Long,
)
